widget/edit.html

```diff
@@ -75,16 +75,20 @@ <h3 class="modal-title col-md-9">Edit Funnel Chart</h3> <button type="button" cl
                             data-form-name="'editFunnelWidgetForm'">
                         </div>
                     </div>
+                    <div id="customModuleField" 
+                    data-ng-class="{ 'has-error': editFunnelWidgetForm.customModuleField.$invalid && editFunnelWidgetForm.customModuleField.$touched }" >
+                        <label class="control-label" >Select Field
+                            <span class="text-danger">*</span>
+                        </label>
+                        <span data-uib-tooltip="Select the Field which has the data, only json type fields will be available in the dropdown"
```

Comment:
Select the Field which has the data, only json type fields will be available in the dropdown --> Select a JSON type field